### PR TITLE
Update to upstream release v0.1.39

### PR DESCRIPTION
**Pull request by vomba**
# Changes Description

- Update to latest release from upstream:
  - This fixes some problems with building images for aws
- Update gloss vars to not fails tests with unattended upgrades change

### CODE REVIEW

**Review by Zash — COMMENTED** [review]
I tried building an image for OpenStack and k8s 1.29.7 with this but it failed (doing it again while capturing logs).

Has anyone else tried building an image?

File: images/capi/ansible/roles/sysprep/tasks/debian.yml [hunk 1]
```diff
@@ -75,25 +80,6 @@
...
-- name: Get installed packages
-  ansible.builtin.package_facts:
-
-- name: Disable unattended upgrades if installed
-  ansible.builtin.systemd:
-    name: unattended-upgrades
-    state: stopped
-    enabled: false
-  when: "'unattended-upgrades' in ansible_facts.packages"
-
```
Comment (Zash) [inline images/capi/ansible/roles/sysprep/tasks/debian.yml:96]: Looks like there are tests that check that these are disabled and fail the build

<details><summary>log snippet</summary>

```
    qemu:         {
    qemu:             "duration": 14848565,
    qemu:             "err": null,
    qemu:             "expected": [
    qemu:                 "false"
    qemu:             ],
    qemu:             "found": [
    qemu:                 "true"
    qemu:             ],
    qemu:             "human": "Expected\n    \u003cbool\u003e: true\nto equal\n    \u003cbool\u003e: false",
    qemu:             "meta": null,
    qemu:             "property": "enabled",
    qemu:             "resource-id": "apt-daily-upgrade.timer",
    qemu:             "resource-type": "Service",
    qemu:             "result": 1,
    qemu:             "successful": false,
    qemu:             "summary-line": "Service: apt-daily-upgrade.timer: enabled:\nExpected\n    \u003cbool\u003e: true\nto equal\n    \u003cbool\u003e: false",
    qemu:             "test-type": 0,
    qemu:             "title": ""
    qemu:         },
    qemu:         {
    qemu:             "duration": 7506652,
    qemu:             "err": null,
    qemu:             "expected": [
    qemu:                 "false"
    qemu:             ],
    qemu:             "found": [
    qemu:                 "true"
    qemu:             ],
    qemu:             "human": "Expected\n    \u003cbool\u003e: true\nto equal\n    \u003cbool\u003e: false",
    qemu:             "meta": null,
    qemu:             "property": "running",
    qemu:             "resource-id": "apt-daily-upgrade.timer",
    qemu:             "resource-type": "Service",
    qemu:             "result": 1,
    qemu:             "successful": false,
    qemu:             "summary-line": "Service: apt-daily-upgrade.timer: running:\nExpected\n    \u003cbool\u003e: true\nto equal\n    \u003cbool\u003e: false",
    qemu:             "test-type": 0,
    qemu:             "title": ""
    qemu:         },
```

</details>

**Review by Zash — CHANGES_REQUESTED** [review]
```
==> qemu: Uploading goss tests...
    qemu: Uploading vars file packer/goss/goss-vars.yaml
    qemu: Inline variables are --vars-inline '{"ARCH":"amd64","OS":"ubuntu","OS_VERSION":"2204","PROVIDER":"qemu","containerd_version":"1.7.20","kubernetes_cni_deb_version":"","kubernetes_cn>
    qemu: Uploading Dir packer/goss
    qemu: Creating directory: /tmp/goss/goss
==> qemu: 
==> qemu: 
==> qemu: 
==> qemu: Running goss tests...
==> qemu: Running GOSS validate command: cd /tmp/goss && sudo  /tmp/goss-0.3.16-linux-amd64 --gossfile goss/goss.yaml --vars /tmp/goss/goss-vars.yaml --vars-inline '{"ARCH":"amd64","OS":"ubu>
    qemu: {
    qemu:     "results": [
=== SNIP ===
    qemu:     "summary": {
    qemu:         "failed-count": 4,
    qemu:         "summary-line": "Count: 60, Failed: 4, Duration: 0.318s",
    qemu:         "test-count": 60,
    qemu:         "total-duration": 317702975
    qemu:     }
    qemu: }
==> qemu: Provisioning step had errors: Running the cleanup provisioner, if present...
==> qemu: Deleting output directory...
Build 'qemu' errored after 13 minutes 13 seconds: Error running Goss: goss non-zero exit status

==> Wait completed after 13 minutes 13 seconds

==> Some builds didn't complete successfully and had errors:
--> qemu: Error running Goss: goss non-zero exit status

==> Builds finished but no artifacts were created.
```

Probably need to find and tweak these tests.

File: images/capi/ansible/roles/sysprep/tasks/debian.yml (hunk 1 above)
Comment (vomba) [reply images/capi/ansible/roles/sysprep/tasks/debian.yml:96]: Could be that there is openstack specific tests that need to be changed too

**Review by Zash — APPROVED** [review]
Image for openstack builds fine. Going to do a quick smoketest to ensure it boots but assuming it has been tested elsewhere I assume it should work fine.

**Review by viktor-f — APPROVED** [review]
I have not tested, but assuming that the smoketest from @Zash works as expected, then I'm ok with this. Thanks for updating this.

**Pull request merged**